vagrant/logs.collect.yml

```diff
@@ -0,0 +1,8 @@
+---
+- hosts: cluster
```

Comment:
This should be "all". Otherwise we won't have logs from client machines.

Reply:
Done